### PR TITLE
OD-5 [Feature] LFD Accesibility

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-5 https://weboo.atlassian.net/browse/OD-5

## Description

- Added accesibility for all lfd components

## Screenshots/screencasts
http://somup.com/crnno4oL3u
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii 
## Notes
Previously reverted PR https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/526

### CODE REVIEW

File: js/layout-javascript/small-card-code.js [hunk 1]
```diff
@@ -534,7 +598,18 @@ DynamicList.prototype.attachObservers = function() {
     .on('hide.bs.collapse', '.small-card-filters-panel .panel-collapse', function() {
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
-    .on('click', '.dynamic-list-add-item', function() {
+    .on('keydown', '.small-card-filters-panel', function(event) {
```
Comment (tonytlwu) [inline js/layout-javascript/small-card-code.js:601]: Why does this not need the `click` event?

File: js/layout-javascript/simple-list-code.js [hunk 2]
```diff
@@ -458,6 +519,13 @@ DynamicList.prototype.attachObservers = function() {
     .on('hide.bs.collapse', '.simple-list-filters-panel .panel-collapse', function() {
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
+    .on('keydown', '.simple-list-filters-panel', function(event) {
```
Comment (tonytlwu) [inline js/layout-javascript/simple-list-code.js:522]: Why does this not need the `click` event?

**Comment by AndrRyaz** [comment]
@tonytlwu hi, fixed

File: js/layout-javascript/simple-list-code.js [hunk 3]
```diff
@@ -480,6 +548,7 @@ DynamicList.prototype.attachObservers = function() {
     .on('click', '.simple-list-comment-close-panel', function() {
```
Comment (tonytlwu) [inline js/layout-javascript/simple-list-code.js:548]: Why does this not need `keydown` event? like the `.news-feed-comment-close-panel` selector needed?

File: js/layout-javascript/simple-list-code.js [hunk 4]
```diff
@@ -458,6 +519,13 @@ DynamicList.prototype.attachObservers = function() {
...
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
+    .on('click keydown', '.simple-list-filters-panel', function(event) {
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
+
+      $(event.target).find('.collapse').collapse('toggle');
+    })
     .on('click', '.simple-list-comment-holder', function(event) {
```
Comment (tonytlwu) [inline js/layout-javascript/simple-list-code.js:529]: Why does this not need `keydown` event like the `.news-feed-comment-holder` selector needed?

Comment (AndrRyaz) [reply js/layout-javascript/simple-list-code.js]: @tonytlwu fixed, i dont understand why this code was lost after the revert.

File: js/layout-javascript/simple-list-code.js (hunk 3 above)
Comment (AndrRyaz) [reply js/layout-javascript/simple-list-code.js]: @tonytlwu, fixed

**Review by tonytlwu — COMMENTED** [review]
@sofiiakvasnevska The code looks good so far. Please put this on development for QA to test. The scope of this change is quite large and it has been a huge blocker whenever a mistake is made. I want QA to test everything that is clickable on all the layouts and device types to confirm it's all working before approving this.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: All clickable LFD features in the app
Verify: All clickable features are working correctly

Squash and merge: OD-5 [Feat] Improve web accessibility

**Pull request merged**